scripts/exregional_run_met_pcpcombine.sh

```diff
@@ -246,7 +246,6 @@ fi
 #
 export MET_INSTALL_DIR
```

Comment:
@natalie-perlin Should `MET_INSTALL_DIR` and `METPLUS_PATH` also be removed from this and the other `exregional_...` scripts since they're being removed from `config_defaults.yaml`?

Reply:
Yes, they would be removed, and instead replaced by loading met and metplus modules.

Reply:
@natalie-perlin Could you remove `MET_INSTALL_DIR` and `METPLUS_PATH` from this and the other `exregional_...` scripts?  Thanks.

Reply:
@natalie-perlin Great, thanks!  Approving now.